src/main/java/com/skankhunt220/api/controller/UserController.java

```diff
@@ -16,31 +20,28 @@
 public class UserController {
     private final UserService userService;
 
-    @GetMapping("/allUsers")
-    public List<UserDto> getAllUsers() throws IOException {
-        List<UserDto> listDto = new ArrayList<>();
-        userService.readAll().forEach(user -> listDto.add(UserTransformer.transformIntoUserDto(user)));
-
-        return listDto;
+    @GetMapping(value = "/{spreadsheetId}", params = {"page", "size"})
```

Comment:
You use **/{spreadsheetId}** in all methods, so move it in main link as a required parameter, like: `@RequestMapping("/api/{spreadsheetId}")`